.github/workflows/label-pr.yml

```
@@ -0,0 +1,75 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This workflow utilizes https://github.com/actions/github-script in GitHub Actions to apply labels to pull requests.
+#
+
+name: Label PR
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
```

```suggestion
    permissions:
      contents: read
      pull-requests: write
```

Possibly? Just taking reference from: https://github.com/actions/labeler#create-workflow

It's also possible to try it with the GITHUB_TOKEN

---

Or try `permissions: read-all|write-all` as mentioned in that link, just to see if it's a permissions issue.

---

@markmandel, please merge this PR. I was able to reproduce the error in my fork by not adding the write permission to the pull_request and issues - https://github.com/Kalaiselvi84/agones/actions/runs/5185413754/jobs/9345294118?pr=274 
You can adjust the permission in Agones if there is any issues. WDYT?

---

Sounds good! Merging!